Sources/Snowplow/Tracker/CrossDeviceParameterConfiguration.swift

```diff
@@ -0,0 +1,47 @@
+//  Copyright (c) 2013-2023 Snowplow Analytics Ltd. All rights reserved.
+//
+//  This program is licensed to you under the Apache License Version 2.0,
+//  and you may not use this file except in compliance with the Apache License
+//  Version 2.0. You may obtain a copy of the Apache License Version 2.0 at
+//  http://www.apache.org/licenses/LICENSE-2.0.
+//
+//  Unless required by applicable law or agreed to in writing,
+//  software distributed under the Apache License Version 2.0 is distributed on
+//  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+//  express or implied. See the Apache License Version 2.0 for the specific
+//  language governing permissions and limitations there under.
+
+import Foundation
+
+/// 
+/// Configuration object for ``TrackerController/decorateLink``
+///
+/// Whether to include the following values when decorating a link:
+/// - `sessionId`: Value of ``SessionController.sessionId``
+/// - `subjectUserId`: Value of ``Subject.userId``
+/// - `sourceId`: Value of ``Tracker.appId``
+/// - `platform`: Value of ``Tracker.platform``
+/// - `reason`:  Optional identifier/information for cross-navigation
+///
+/// ``sourceId`` and ``sessionId`` are enabled by default.
+public class CrossDeviceParameterConfiguration : NSObject {
```

Comment:
Can we add `@objc` annotations on the class as well as the properties and constructor? (also please API doc comments for the properties 🥺)